src/Sql/dbo/Views/SsoConfigView.sql

```diff
@@ -1,6 +1,8 @@
 ﻿CREATE VIEW [dbo].[SsoConfigView]
 AS
 SELECT
-    *
+    SSO.*
 FROM
-    [dbo].[SsoConfig]
+    [dbo].[SsoConfig] SSO
+INNER JOIN
```

Comment:
I am not sure I understand the need for the join in this view? Other views are mostly just basic `select * from [table]`. The joins can occur in the sproc for `SsoConfig_GetByIdentifier`

Reply:
Moved join to appropriate sproc